packages/support/src/helpers.php

```diff
@@ -170,6 +170,18 @@ function generate_search_column_expression(string $column, ?bool $isSearchForced
             };
         }
 
+        $column = match ($driverName) {
```

Comment:
Should this be combined with the `match` below it maybe?

Also, does this change only need to take place when `lower()` is used, or is it also without `lower()`?